### ISSUE TITLE
bug? when vec has no elements, tera will render one blank row

### ISSUE THREAD

**Issue by daogangtang** [Keats/tera#36]
in html, I need to display a table, such as: 

白拿活动申请结果表
序号 	蜥客名 	电话 	艺术品id 	艺术品名称 	时间
1 					

when no data, It display one line yet.
PS: I used `loop.index` as first column.

this is very easy to check.

**Comment by Keats** [comment]
can you paste the html you're using?

**Comment by daogangtang** [comment]
```
<body>
    <h1>活动申请结果表</h1>

    <div class="body">
        <table>
            <thead>
                <tr>
                    <th>序号</th>
                    <th>蜥客名</th>
                    <th>电话</th>
                    <th>艺术品id</th>
                    <th>艺术品名称</th>
                    <th>时间</th>
                </tr>
            </thead>
            <tbody>
                {% for item in baina_list %}
                <tr>
                    <td>{{loop.index}}</td>
                    <td>{{item.nick_name}}</td>
                    <td>{{item.phone}}</td>
                    <td>{{item.art_id}}</td>
                    <td>{{item.art_name}}</td>
                    <td>{{item.apply_time}}</td>
                </tr>
                {% endfor %}
            </tbody>
        </table>
    </div>


</body>
```

Yes, you have a normal forloop with 0 items in the Vec but it still iterates once?  At lease show one line.

**Comment by Keats** [comment]
Fixed in master, i'll push a new version soon

**Comment by daogangtang** [comment]
nice

**Comment by Keats** [comment]
Released in 0.1.2
